src/main/java/seedu/binbash/Parser.java

```
@@ -92,6 +98,32 @@ private Command parseAddCommand(String userInput) throws InvalidFormatException
                     itemCostPrice);
     }
 
+    private Command parseRestockCommand(String userInput) throws InvalidFormatException {
+        Matcher matcher = RestockCommand.COMMAND_FORMAT.matcher(userInput);
+        if (!matcher.matches()) {
+            throw new InvalidFormatException("Restock command is not properly formatted!");
+        }
+        String itemName = matcher.group("itemName");
+        int restockQuantity = Integer.parseInt(
+                Objects.requireNonNullElse(matcher.group("restockQuantity"), "0").strip()
```

Since restockQuantity is a compulsory parameter, I feel it would be better to throw an error when null is obtained, as defaulting it to 0 is something we do for optional parameters. (Same for L121) What are your thoughts on this?


(Although I understand adding and subtracting by 0 does nothing to itemQuantity in the end, i think it would still be good practice to enforce proper error handling of incorrect user inputs)